priam/src/main/java/com/netflix/priam/defaultimpl/PriamGuiceModule.java

```diff
@@ -84,6 +92,8 @@ protected void configure() {
         } else {
             bind(IMembership.class).to(AWSMembership.class).asEagerSingleton();
         }
+
+        bind(String.class).annotatedWith(Names.named("awsRoleAssumptionARN")).toInstance(priamConfiguration.getCassandraConfiguration().getSdbRoleAssumptionArn());
```

Comment:
Sorry, should have caught this earlier.  Guice can't bind `null` to a value, and if no ARN was provided then it would be null.  Mabye bind to `Optional<String>` instead?